kubectl-fdb/cmd/remove_process_group.go

```diff
@@ -127,118 +134,78 @@ kubectl fdb -n default remove process-groups -c cluster --process-class="statele
 }
 
 type replaceProcessGroupsOptions struct {
-	clusterLabel      string
-	processClass      string
-	withExclusion     bool
-	wait              bool
-	removeAllFailed   bool
-	useProcessGroupID bool
+	withExclusion   bool
+	wait            bool
+	removeAllFailed bool
 }
 
 // replaceProcessGroups adds process groups to the removal list of their respective clusters, and returns a count of
 // the number of removed processGroups (pods) and any encountered error.
 // If clusterName is specified, it will ONLY do so for the specified cluster.
 // If processClass is specified, it will ignore the given ids and remove all processes in the given cluster whose pods
 // have a processClassLabel matching the processClass.
-func replaceProcessGroups(cmd *cobra.Command, kubeClient client.Client, clusterName string, ids []string, namespace string, opts replaceProcessGroupsOptions) (int, error) {
-	if len(ids) == 0 && !opts.removeAllFailed && opts.processClass == "" {
-		return 0, nil
-	}
-
-	// cross-cluster logic: given a list of Pod names, we can look up the FDB clusters by pod label, and work across clusters
-	if !opts.useProcessGroupID && clusterName == "" {
-		pgsByCluster, err := fetchProcessGroupsCrossCluster(kubeClient, namespace, opts.clusterLabel, ids...)
-		if err != nil {
-			return 0, err
-		}
-		totalRemoved := 0
-		for cluster, pgs := range pgsByCluster {
-			err := replaceProcessGroupsFromCluster(kubeClient, cluster, pgs, namespace, opts.withExclusion, opts.wait, opts.removeAllFailed)
-			if err != nil {
-				return 0, err
-			}
-			totalRemoved += len(pgs)
-			cmd.Printf("Cluster %v/%v:\n", namespace, cluster.Name)
-			cmd.Printf("removed %v (exclude: %t)\n", pgs, opts.withExclusion)
-		}
-		return totalRemoved, nil
+func replaceProcessGroups(cmd *cobra.Command, kubeClient client.Client, pgSelectOpts processSelectionOptions, opts replaceProcessGroupsOptions) (int, error) {
```

Comment:
I believe in most cases we tended to rather have longer names instead of abbreviations like `pg`.

Reply:
Ack! wasn't sure here since it gets lengthly, I'll start defaulting to longer then 😄